wal/src/manager.rs

```diff
@@ -119,12 +119,28 @@ pub mod error {
     define_result!(Error);
 }
 
-use common_types::{MAX_SEQUENCE_NUMBER, MIN_SEQUENCE_NUMBER};
+use common_types::{table::TableId, MAX_SEQUENCE_NUMBER, MIN_SEQUENCE_NUMBER};
 pub use error::*;
 
 pub type RegionId = u64;
 pub const MAX_REGION_ID: RegionId = u64::MAX;
 
+/// Used to wals of the table.
```

Comment:
Used to describe where to store the wals of the table.

Reply:
Done.